tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/BrowserBased/BrowserAdditionalInformation/Supplier Edit BrowserAdditionalInformation Validation.feature

```diff
@@ -12,7 +12,10 @@ Background:
         | Sln1       | MedicOnline    | Sup 1      |
     And solutions have the following details
         | SolutionId | SummaryDescription             | FullDescription   | ClientApplication                                                                               |
-        | Sln1       | An full online medicine system | Online medicine 1 | { "AdditionalInformation": "Some Info", "ClientApplicationTypes": [], "BrowsersSupported": [] } |
+        | Sln1       | An full online medicine system | Online medicine 1 | { "AdditionalInformation": "Some Info", "ClientApplicationTypes": [], "BrowsersSupported": [] } |        
+    And Framework Solutions exist
```

Comment:
Could you make this step definition lower case so it becomes `And framework solutions exist`?

There are some inconsistencies in the casing of step definitions in this solution but this is the preferred format.

I know this request is a bit of a chore but at least you should just be able to do a find and replace! 😃

Reply:
Updated format of `Given` feature step `And framework solutions exist `